cmd/snap-seccomp/main.go

```diff
@@ -388,6 +388,18 @@ var seccompResolver = map[string]uint64{
 	"NETLINK_RDMA":           C.NETLINK_RDMA,
 	"NETLINK_CRYPTO":         C.NETLINK_CRYPTO,
 	"NETLINK_INET_DIAG":      C.NETLINK_INET_DIAG, // synonymous with NETLINK_SOCK_DIAG
+
+	// man 2 ptrace
+	"PTRACE_ATTACH":     syscall.PTRACE_ATTACH,
+	"PTRACE_DETACH":     syscall.PTRACE_DETACH,
```

Comment:
I'm a little surprised that you didn't need PTRACE_CONT here. It is fine to not have it since we are only concerned about Breakpad, but would it make sense to include it here and in the policy below for future-proofing? Not a blocker.